clients/lsp-javascript.el

```diff
@@ -892,11 +898,23 @@ name (e.g. `data' variable passed as `data' parameter)."
     (when lsp-javascript-update-inlay-hints-on-scroll
       (setf window-scroll-functions (delete #'lsp-javascript-update-inlay-hints-scroll-function window-scroll-functions))))))
 
+(defun lsp-clients-typescript-project-ts-server-path ()
+  (f-join (lsp-workspace-root) "node_modules" "typescript" "lib" "tsserver.js"))
+
+(defun lsp-clients-typescript-server-path ()
```

Comment:
I wish we somehow log this value.

Reply:
I'd be happy to add something more explicit, but it's indirectly already logged (see the `--tsserver-path` argument):

```
Command "/Users/shep/.emacs.d/.cache/lsp/npm/typescript-language-server/bin/typescript-language-server --tsserver-path /Users/shep/.emacs.d/.cache/lsp/npm/typescript/bin/tsserver --stdio" is present on the path.
```